backend/main.py

```
@@ -32,38 +30,74 @@ class DevRAIApplication:
+                            self.discord_bot.start(settings.discord_bot_token)
+                        )
+                except Exception as e:
+                    logger.exception("Discord startup failed: %s",e)
+                    self.discord_bot = None
+            
+            # Full Mode            
+            if settings.code_intelligence_enabled:
+                # Weaviate
+                try:
+                    await self.test_weaviate_connection()
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)